depends/packages/native_rust.mk

```diff
@@ -13,6 +13,7 @@ $(package)_sha256_hash_freebsd=36a14498f9d1d7fb50d6fc01740960a99aff3d4c4c3d2e4ff
 # "darwin", in which case we assume x86_64-apple-darwin.
 $(package)_rust_target_x86_64-pc-linux-gnu=x86_64-unknown-linux-gnu
 $(package)_rust_target_x86_64-w64-mingw32=x86_64-pc-windows-gnu
+$(package)_rust_target_x86_64-unknown-freebsd12.2=x86_64-unknown-freebsd
```

Comment:
Hmm... if FreeBSD includes its full version in the host string (like Darwin does), then we should do the same thing we do for Darwin (`findstring freebsd` to select this inside `define rust_target`).

Reply:
Addressed in https://github.com/zcash/zcash/pull/4937/commits/a0c1ba16db68f9518cf6d0e1f421fa98e4aefb31.